Sources/AEPOptimize/Optimize+PublicAPI.swift

```diff
@@ -22,8 +22,9 @@ public extension Optimize {
     /// - Parameter decisionScopes: An array of decision scopes.
     /// - Parameter xdm: Additional XDM-formatted data to be sent in the personalization request.
     /// - Parameter data: Additional free-form data to be sent in the personalization request.
-    @objc(updatePropositions:withXdm:andData:)
-    static func updatePropositions(for decisionScopes: [DecisionScope], withXdm xdm: [String: Any]?, andData data: [String: Any]? = nil) {
+    /// - Parameter completion: An optional callback to be used by consumer
```

Comment:
can you add what is returned in the callback? 
`- Parameter completion: Optional completion handler invoked with errors, if any`

Reply:
Done